adback-solution-to-adblock/includes/class-ad-back-updator.php

```diff
@@ -136,6 +136,25 @@ public static function isRewriteRouteEnabled()
         return (bool) get_option('permalink_structure');
     }
 
+    public static function isRewriteRulesConflictWithEndpoints($endpoint)
+    {
+        require_once( ABSPATH . 'wp-includes/class-wp-rewrite.php' );
+
+        if (!$rules = get_option('rewrite_rules')) {
```

Comment:
Does it directly output an array or do you have to unserialize it ?

Reply:
`get_option()` return unserialized array yes